x-pack/plugin/migrate/src/main/java/org/elasticsearch/xpack/migrate/action/ReindexDataStreamAction.java

```diff
@@ -47,20 +43,11 @@ public class ReindexDataStreamAction extends ActionType<ReindexDataStreamAction.
      * The version before which we do not support writes in the _next_ major version of Elasticsearch. For example, Elasticsearch 10.x will
      * not support writing to indices created before version 9.0.0.
```

Comment:
This comment needs to be removed from here.

Reply:
Done :-)